strategies/snapchat.js

```diff
@@ -0,0 +1,16 @@
+const SnapchatStrategy = require('passport-snapchat').Strategy;
+
+module.exports = new SnapchatStrategy({
+    clientID: process.env.SNAPCHAT_CLIENT_ID,
+    clientSecret: process.env.SNAPCHAT_CLIENT_SECRET,
+    callbackURL: "http://localhost:3000/auth/snapchat/callback",
+    profileFields: ['id', 'displayName', 'bitmoji']
+  },
+  async (accessToken, refreshToken, profile, done) => {
+    const tokens = {
+      accessToken: accessToken,
```

Comment:
node runs with ES6 (javascript with some syntax sugar), so you can actually declare the object like this
```
const tokens = {
  accessToken,
  refreshToken,
};
```

[Reference](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand) to the shorthand. Basically your variable name becomes the name on the object's key. So if the variable name happens to be the same as what you want to name the key then you can declare it like this

Reply:
cool, good to know!